docs/offline-environment.md

```diff
@@ -12,7 +12,7 @@ following artifacts in advance from another environment where has access to the
 
 Then you need to setup the following services on your offline environment:
 
-* a HTTP reverse proxy/cache/mirror to serve some static files (zips and binaries)
+* an HTTP reverse proxy/cache/mirror to serve some static files (zips and binaries)
```

Comment:
The original seems good for me.

Reply:
https://www.pristineword.com/grammar-an-http/
ok seems like @Fish-pro is right